test/isArray_test.js

```diff
@@ -0,0 +1,65 @@
+goog.module('datalayerhelper.helper.testing.isArray_');
+goog.setTestOnly();
+
+const {isArray_} = goog.require('helper');
+
+describe('The `isArray_` function of helper', () => {
+  /**
+   * Ensure that plain.IsPlainObject recognizes value
```

Comment:
Shouldn't this be the "isArray_" function?

Reply:
Yes it should be